go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/operator-framework/operator-controller
 
-go 1.24.6
```

Comment:
Don't change this, it will be dealt with when the upstream is synced (the bumper touches this, and we don't want to carry this change along)